R/prevalence.R

```diff
@@ -0,0 +1,94 @@
+#' @title Taxa Prevalence 
+#' @description Simple prevalence measure.
+#' @param x A vector, data matrix or \code{\link{MicrobiomeExperiment}} object
```

Comment:
use the S4 method dispatch system. Include a method for `SummarizedExperiment` and `ANY`. If you need suggestions on how to do this, let me know.

Reply:
Just to check - you propose that I will make a PR to `SE` directly? Should be fine.

Reply:
I am not sure how you mean this. Not to `SummarizedExperiment` package, but to `mia`

I have this in mind:
```
setGeneric("getPrevalence", signature = "x",
           function(x, ...)
               standardGeneric("getPrevalence"))

setMethod("getPrevalence", signature = c(x = "ANY"),
    function(x, ...){
        # do vector or matrix stuff here       
    }
)

setMethod("getPrevalence", signature = c(x = "SummarizedExperiment"),
    function(x, ...){
        # do SE stuff here and subest to matrix or vector
        mat <- some_stuff()
        getPrevalence(mat, ...)
    }
)
```